### PR TITLE
Add native_tokens method to Output

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Add native_tokens method to Output

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: bee-message/src/output/mod.rs [hunk 1]
```diff
@@ -101,7 +101,7 @@ impl Output {
         }
     }
 
-    ///
+    /// Return the IOTA amount of an `Output`.
```
Comment (thibault-martinez) [inline bee-message/src/output/mod.rs]:
```suggestion
    /// Returns the amount of an `Output`.
```

Can also be Shimmer amount

File: bee-message/src/output/mod.rs [hunk 2]
```diff
@@ -111,4 +111,15 @@ impl Output {
             Self::Nft(output) => output.amount(),
         }
     }
+
+    /// Return the native tokens of an `Output`, if any.
+    pub fn native_tokens(&self) -> Option<Vec<NativeToken>> {
```
Comment (thibault-martinez) [inline bee-message/src/output/mod.rs]: I think I would return a slice here, if you need a Vec you can do it yourself but for most usecases a slice would be enough

File: bee-message/src/output/mod.rs [hunk 3]
```diff
@@ -111,4 +111,15 @@ impl Output {
             Self::Nft(output) => output.amount(),
         }
     }
+
+    /// Return the native tokens of an `Output`, if any.
```
Comment (thibault-martinez) [inline bee-message/src/output/mod.rs]:
```suggestion
    /// Returns the native tokens of an `Output`, if any.
```

**Review by thibault-martinez — APPROVED** [review]

**Review by Adam-Gleave — APPROVED** [review]

**Review by Alex6323 — APPROVED** [review]

**Review by grtlr — APPROVED** [review]

**Review by pvdrz — APPROVED** [review]

**Comment by grtlr** [comment]
That must be the PR with the most approvals 😁.

**Comment by Thoralf-M** [comment]
@grtlr you missed https://github.com/iotaledger/bee/pull/1053 :trollface:

**Pull request merged**